src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/resource/ClientsController.kt

```diff
@@ -29,6 +31,44 @@ class ClientsController(
   fun editClient(@PathVariable clientId: String, @RequestBody clientDetails: ClientCredentialsUpdateRequest) {
     clientService.editClientCredentials(clientId, clientDetails)
   }
+
+  @GetMapping("clients/client-credentials/{clientId}/view")
```

Comment:
swagger docs required here?

Reply:
No, these end points are only intended for use by the new UI